src/openscm_units/_unit_registry.py

```diff
@@ -142,12 +142,13 @@
     <Quantity(0.823529412, 'N')>
 
 """
+import importlib
 import math
-from os import path
 
 import pandas as pd
 import pint
 
+from . import data  # pylint:disable=no-name-in-module # no idea why this is failing
```

Comment:
@mikapfl any idea why pylint doesn't like this pattern?

Reply:
I am confused, If I run `make checks` locally with the `pylint:disable…` removed, I don't get any error.

Reply:
Hmm ok fixed for me too now, all good